packages/moonstone/Panels/Header.module.less

```diff
@@ -90,6 +88,7 @@
 	// Compact Header
 	&.compact {
 		height: @moon-header-height-small;
+		height: 87px;
```

Comment:
Guessing this wasn't supposed to be here.

Reply:
Does seem out of place, doesn't it.